.fides/fides.toml

```diff
@@ -7,7 +7,7 @@ db = "fides"
 test_db = "fides_test"
 
 [credentials]
-app_postgres = {connection_string="postgresql+psycopg2://postgres:fides@fides-db:5432/fides_test"}
+app_postgres = {connection_string="postgresql+psycopg2://postgres:fides@fides-db:5432/fides"}
```

Comment:
Running `fides scan dataset db --credentials-id app_postgres` locally should now scan the application db not the test db